src/core/public/legacy/legacy_service.ts

```diff
@@ -34,28 +47,29 @@ export interface LegacyPlatformParams {
  * setup either the app or browser tests.
  */
 export class LegacyPlatformService {
+  private bootstrapModule?: BootstrapModule;
+  private targetDomElement?: HTMLElement;
+
   constructor(private readonly params: LegacyPlatformParams) {}
 
-  public setup(core: CoreSetup) {
+  public async setup({ core }: SetupDeps) {
```

Comment:
I did this for consistency, since all the other services accept an option bag. Now that I'm reading this again, I think I'll put it back since CoreSetup is already a bag of services.